### ISSUE TITLE
Make use of github actions

### ISSUE THREAD

**Issue by jwillemsen** [objectcomputing/OpenDDS#1854]
Currently for CI azure-pipelines and travis-ci are used. Worked a little bit on github actions for AXCIOMA/TAOX11/ATCD, testing shows it is faster and we have much more workers speeding up things.

**Comment by mitza-oci** [comment]
Yes, it makes sense to go that way.  We have GitHub Actions in this repository on the gh-pages branch, and I've experimented with it for some smaller repos.  Especially since the way it works seems to be very similar to Azure Pipelines it shouldn't be difficult to migrate.

**Comment by jwillemsen** [comment]
ACE/TAO, CIAO, DAnCE are now using github actions. Also for the axcioma/taox11/ciaox11/dancex11/ridl/r2corba repositories under RemedyIT we are using github actions. ACE/TAO has 30 checks but these are all started in parallel when a new pull request is made, on azure we had a max of 10 jobs in parallel

**Comment by iguessthislldo** [comment]
Closing because we added building and testing in Actions in #2118.

**Comment by iguessthislldo** [comment]
Thanks, I meant to hit "Close issue with comment", but I must have just hit "Comment".
